bin/common.sh

```diff
@@ -18,6 +18,13 @@
 # Referred from Apache Submarine's common.sh implementation
 # bin/common.sh
 
+GRAVITINO_VERSION=GRAVITINO_VERSION_PLACEHOLDER
+if [[ "$GRAVITINO_VERSION" == *_VERSION_PLACEHOLDER ]]; then
+  echo "GRAVITINO_VERSION is not set. Please make sure you are running the script from the distribution/package/bin and before running the script, run './gradle clean build -x test compileDistribution'"
+  exit 1
```

Comment:
Please update the prompt message:
```
No GRAVITINO_VERSION was found, you may need to:
1. Run the gravitino.sh script on the compiled Gravitino.
2. Run the gravitino.sh script in the release package.
```

Reply:
Thank you for the feedback. I will make the updates as required:
- Update the error message for startup failures
- Adjust the version check
- Revert the root build.gradle.kts files

Reply:
"Run the gravitino.sh script in the release package." this may not be the best instructions, as an ASF release package is source, not binary.

Reply:
What does 'release package' mean? Should it instead be referred to as a 'distribution package'?

Reply:
@justinmclean The problem is that there are some users who run gravitino.sh scripts without compiling, causing them to fail. We want to give them some tips. Do you have any better suggestions?

Reply:
Ask them to use a binary distribution or compile it first? i.e:
1. Run the gravitino.sh script in a compiled version of Gravitino.
2. Run the gravitino.sh script from a binary release package.